cmd/root.go

```diff
@@ -32,6 +32,11 @@ var (
 		otel                   bool
 		otelAddress            string
 
+		datastoreMaxIdleConnections    int
```

Comment:
maybe `connectionPoolMaxOpenConnections`?

Reply:
I thought this could be a little long but I think you're right this is more precise.

Reply:
Went with the struct